StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Saved Payment Method Screen/Vertical Saved Payment Method Screen/PaymentMethodRowButton.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  PaymentMethodRowButton.swift
+//  StripePaymentSheet
+//
+//  Created by Nick Porter on 5/9/24.
+//
+
+import Foundation
+@_spi(STP) import StripeCore
+@_spi(STP) import StripePaymentsUI
+@_spi(STP) import StripeUICore
+import UIKit
+
+protocol PaymentMethodRowDelegate: AnyObject {
+    func didSelectButton(_ button: PaymentMethodRowButton)
+    // TODO(porter) Add did delete and did update
+}
+
+final class PaymentMethodRowButton: UIView {
```

Comment:
Same, omitted snapshot tests for this view and relied on `VerticalSavedPaymentOptionsViewControllerSnapshotTests`.